tests/common/devices.py

```diff
@@ -1608,6 +1647,21 @@ def __init__(self, ansible_adhoc, hostname):
         """
         self.sonichost = SonicHost(ansible_adhoc, hostname)
         self.asics = [SonicAsic(self.sonichost, asic_index) for asic_index in range(self.sonichost.facts["num_asic"])]
+        self.critical_services_tracking_list()
+
+    def critical_services_tracking_list(self):
+        """Get the list of services running on the DUT
+           The services on the sonic devices are:
+              - services running on the host 
+              - services which are replicated per asic
+            Returns:
+            [list]: list of the services running the device
+        """
+        service_list = []
+        service_list+= self._DEFAULT_SERVICES
+        for asic in self.asics:
+            service_list += asic.get_critical_services()
+        self.sonichost.reset_critical_services_tracking_list(service_list)
```

Comment:
There's already a setter for critical_services. Is 'reset_critical.." API really requried.

Reply:
Yes this is required, for `MultiAsicSonicHost` the crtical_services setter is in `SonicHost` class